tools/simple/README.md

```diff
@@ -3,6 +3,10 @@
 `run_simple.sh` is an example script illustrating how the simple import docker image
 can be run locally.
 
-In the `env.list` file, specify the `DC_API_KEY`.
-If using local input and output dirs, specify `INPUT_DIR` and `OUTPUT_DIR` respectively.
-If using GCS input and output dirs, specify `GCS_INPUT_DIR` and `GCS_OUTPUT_DIR` respectively.
+* In the `env.list` file, specify the `DC_API_KEY`.
+* If using GCS:
+  * Run `gcloud beta auth application-default login`.
+  * Specify your cloud project via the `GOOGLE_CLOUD_PROJECT` variable.
+* If using config file and output dirs, specify `CONFIG_FILE` and `GCS_OUTPUT_DIR` (or `OUTPUT_DIR`) respectively.
+* If using local input and output dirs, specify `INPUT_DIR` and `OUTPUT_DIR` respectively.
+* If using GCS input and output dirs, specify `GCS_INPUT_DIR` and `GCS_OUTPUT_DIR` respectively.
```

Comment:
High level thought, can we consolidate to have INPUT_DIR and OUTPUT_DIR and handle checking if it's GCS path or not in the code? So user would

1) Give config_file or
2) give input_dir and output_dir